bower.json

```diff
@@ -28,10 +28,14 @@
     "bignumber.js": "1.4.1",
     "angular-re-captcha": "0.0.3",
     "angulartics": "0.17.1",
-    "URIjs": "1.14.1"
+    "URIjs": "1.14.1",
+    "underscore": "~1.7.0"
```

Comment:
Ok I see. I just got a `_` is undefined error and I assumed this was underscore (funny enough it did fix it). I'll remove this and if I get the previous error again I'll know where to look. Thanks